en/06_Project_Governance/06_Minor_release_policy.md

```diff
@@ -0,0 +1,102 @@
+
+Once the team is ready, stable versions of all supported modules are tagged. A complete changelog is posted to the Silverstripe CMS documentation site along with any relevant upgrade notes.
+
+## Minor release support timeline
+
+Minor release support timeline follow similar phases to major release. But those phases are far more condensed.
+
+### Bug and security fixes
+
+A Silverstripe CMS minor release line enters the *bug and security fixes* phase once it is tagged *stable*.
+
+A minor release in the *bug and security fixes* phase receives:
+- regular patches for bugs at all impact levels
+- security patches for all vulnerabilities.
+
+It does **not** receive:
+- new feature
+- new APIs.
+
+A minor release line stays in the *bug and security fixes* phase until a follow up minor release is tagged.
+
+### Security fixes only
+
+Once the follow up minor release is tagged, the minor release that was in the *bug and security fixes* phase transition to a *security fixes only* phase.
+
+In the *security fixes only* phase, a Silverstripe CMS minor release line only receives patches for high impact security issues defined has having a CVSS of 7.0 or above.
+
+Upon entering the *security fixes only* phase, the end-of-life date for that minor release is scheduled for six months later.
+
+### End-of-life
+
+Six months after entering the *security fixes only*, the minor release goes end-of-life. No more security patches will be shipped for that minor release.
+
+### The last minor release in a major release line
+
+Shortly before tagging a new major release, one last minor release will be tagged for the major release line in active development.
+
+The last minor release in a major release line will have an extended support phases in line with our major release policy:
+- the last minor stays in the *bug and security fixes* phase for one year
+- then it enters the *security fixes only* phase for another year
+- then the minor release goes end of life along with its corresponding major release line.
+
+## Security patch windows
+
+Silverstripe CMS aims to deliver security patches at pre-defined intervals to minimise upgrades woes for Silverstripe CMS projects.
+
+When a vulnerability is [confidentially reported](/contributing/issues_and_bugs/#reporting-security-issues) to the Silverstripe CMS development team and we are confident that the vulnerability is not actively being exploited in the wild, we will wait until the next available security patch window to disclose it and release a patch.
+
+Security patch windows are scheduled to line up with our expected minor release. Our security patch windows are in:
+- January
+- April
+- July
+- October.
+
+Any security fixes that are shipped at launch with a follow up minor release are also backported to minor currently in the minor release in the "bug and security fixes".
+
+## Patch releases
+
+Silverstripe CMS tags patch releases of individual modules whenever relevant bug fixes are merged into the codebase.
```

Comment:
```suggestion
Silverstripe CMS tags patch releases of individual modules whenever relevant bug fixes or other very low risk changes are merged into the codebase.
```
e.g. translations have been tagged as patch releases a few times now.

Reply:
Mix this with Steve's suggestion.